src/coreclr/vm/typehash.cpp

```diff
@@ -167,7 +167,7 @@ static DWORD HashPossiblyInstantiatedType(mdTypeDef token, Instantiation inst)
 }
 
 // Calculate hash value for a function pointer type
-static DWORD HashFnPtrType(BYTE callConv, DWORD numArgs, TypeHandle *retAndArgTypes)
+static DWORD HashFnPtrType(uint32_t sigLen, BYTE callConv, DWORD numArgs, TypeHandle *retAndArgTypes)
```

Comment:
Are we not going to include the `sigLen` in the new hash function?

Reply:
`sigLen` can be different between two different FPs in two different modules based the tokens in the custom mods.

Reply:
I'll remove the parameter.